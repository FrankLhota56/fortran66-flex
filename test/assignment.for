C
C Assignment test
C
         I = 1234
      A(4, 5*I-4) = 1.5
      Call P(4)
      Call P(4) = 1.5E1
        L = .true. .and. .not. .false.
      End
