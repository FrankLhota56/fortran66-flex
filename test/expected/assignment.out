Line     4: ID(i)
Line     4: '='
Line     4: INT_LIT(1234)
Line     5: EOS
Line     5: ID(a)
Line     5: '('
Line     5: INT_LIT(4)
Line     5: ','
Line     5: INT_LIT(5)
Line     5: '*'
Line     5: ID(i)
Line     5: '-'
Line     5: INT_LIT(4)
Line     5: ')'
Line     5: '='
Line     5: REAL_LIT(1.500000)
Line     6: EOS
Line     6: CALL
Line     6: ID(p)
Line     6: '('
Line     6: INT_LIT(4)
Line     6: ')'
Line     7: EOS
Line     7: ID(callp)
Line     7: '('
Line     7: INT_LIT(4)
Line     7: ')'
Line     7: '='
Line     7: REAL_LIT(15.000000)
Line     8: EOS
Line     8: ID(l)
Line     8: '='
Line     8: TRUE
Line     8: AND
Line     8: NOT
Line     8: FALSE
Line     9: EOS
Line     9: END
